### DISCUSSION TITLE
Unable to access subreddit property through my db.

### DISCUSSION THREAD

**Discussion by bottercode** [prisma/prisma#19884] [Q&A] [answered]
![image](https://github.com/prisma/prisma/assets/74648201/d27c244c-c12a-4e5d-a296-a419b4a7869d)

db.subreddit is not accessible.

![image](https://github.com/prisma/prisma/assets/74648201/b0c99ad1-6895-4a6f-b47c-063daa027a8e)
 
*I am using supabase, is it because of supabase that i am unable to access?*

**Comment by ludralph** [comment]
Hi @bottercode 👋 

Did you run `npx prisma generate` to generate the Prisma Client? You may also want to show the code of how you are instantiating the Prisma Client.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by bottercode** [comment]
Instead I ran prisma migrate dev and it gave the following output
![image](https://github.com/prisma/prisma/assets/74648201/78305333-abc7-44b6-a202-5a14ba5813a5)

It gave me some models that I created but rest of them were inaccessible. 
![image](https://github.com/prisma/prisma/assets/74648201/702af436-efb2-47b7-a334-1f42f52b4f66)
(account , session, users is accessible but subreddit is not)

Thank you for the help!

**Reply by ludralph** [reply]
@bottercode, can you please share your schema in text form and the contents of your migration.sql file?

**Comment by bottercode** [comment]
Sure!

*schema.prisma*
https://justpaste.it/8uqz5

---------

*migration.sql*
https://justpaste.it/d2z5t

**Reply by ludralph** [reply, marked as answer]
Hi @bottercode, I've verified that the schema looks okay and the generated migration file looks okay too. Can you maybe restart your IDE and try again? If it doesn't work, you may want to share your repository so I can investigate this further.

**Reply by bottercode** [reply]
How did I not try reloading the window. I do that often. It worked though. A big thanks for the time.

**Reply by nurul3101** [reply]
Thanks for confirming that this issue is resolved now. 🙌 
I'll mark this discussion as answered
